packages/front-end/components/Features/Rule.tsx

```diff
@@ -222,7 +300,7 @@ export const Rule = forwardRef<HTMLDivElement, RuleProps>(
                 }}
               ></div>
             )}
-            {rule.condition && rule.condition !== "{}" && (
+            {hasConditions && (
```

Comment:
Creating this condition is no longer needed as this logic is now only used in this one spot.